### PR TITLE
XFE_v2 Bug

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc/issues/28321

## Description
Fixed an issue where the ***url*** and ***domain*** commands failed for URL's that don't exist in the server.
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py [hunk 1]
```diff
@@ -34,8 +34,13 @@ def ip_report(self, ip: str) -> dict:
...
         return self._http_request('GET', f'/ipr/{ip}')
 
-    def url_report(self, url: str) -> dict:
-        return self._http_request('GET', f'/url/{url}').get('result')
+    def url_report(self, url: str):
+        try:
+            response = self._http_request('GET', f'/url/{url}')
+        except Exception as e:
+            if "Not Found" in str(e):
+                return "Not Found"
```
Comment (yaakovi) [inline Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py:42]: raise the exception again if the condition is not satisfied.
```suggestion
                return "Not Found"
            raise
```

File: Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py [hunk 2]
```diff
@@ -202,7 +207,9 @@ def domain_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, Any
 
     for domain in domains:
         report = client.url_report(domain)
-
+        if report == "Not Found":
+            markdown += f'Domain: {domain} not found'
```
Comment (yaakovi) [inline Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py:211]:
```suggestion
            markdown += f'Domain: {domain} not found\n'

```

File: Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py [hunk 3]
```diff
@@ -249,7 +256,9 @@ def url_command(client: Client, args: Dict[str, str]) -> Tuple[str, dict, Any]:
 
     for url in urls:
         report = client.url_report(url)
-
+        if report == "Not Found":
+            markdown += f'URL: {url} not found'
```
Comment (yaakovi) [inline Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py:260]:
```suggestion
            markdown += f'URL: {url} not found\n'

```

**Review by yaakovi — APPROVED** [review]

**Comment by kirbles19** [comment]
@Bargenish Done.

**Pull request merged**